migrations/__tests__/_all.spec.ts

```diff
@@ -1,9 +1,22 @@
-import { dbFactory } from '@/__tests__/db.factory';
+import { TestDbFactory } from '@/__tests__/db.factory';
 import { PostgresDatabaseMigrator } from '@/datasources/db/postgres-database.migrator';
+import { faker } from '@faker-js/faker';
+import postgres from 'postgres';
 
 describe('Migrations', () => {
-  const sql = dbFactory();
-  const migrator = new PostgresDatabaseMigrator(sql);
+  let sql: postgres.Sql;
+  let migrator: PostgresDatabaseMigrator;
+  const testDbFactory = new TestDbFactory();
+
+  beforeAll(async () => {
+    sql = await testDbFactory.createTestDatabase(faker.string.uuid());
+    migrator = new PostgresDatabaseMigrator(sql);
+  });
+
+  afterAll(async () => {
+    await testDbFactory.dropTestDatabase(sql);
+    await testDbFactory.close();
```

Comment:
This seems to be a common pattern. What do you think about combining the methods into something like `destroyTestDatabase`?

Reply:
Done [e09881a](https://github.com/safe-global/safe-client-gateway/pull/1735/commits/e09881aa7a589b485757217b3a1a9f8f26ae22f8)